x/emissions/migrations/v4/migrate.go

```diff
@@ -156,6 +174,61 @@ func MigrateTopics(
 	return nil
 }
 
+// Deletes all keys in the store with the given keyPrefix `maxPageSize` keys at a time
+func safelyClearWholeMap(store storetypes.KVStore, keyPrefix []byte) {
+	s := prefix.NewStore(store, keyPrefix)
```

Comment:
are we deleting all from each store? Aren't we just attempting to delete topic 5 data?
(remember this will take some more time when running the update on nodes ... worth showing some progress log on the stores being cleaned maybe?)

Reply:
We're deleting everything, I think it's safer.

I don't think it will take as long because there's only a few days of inferences saved as opposed to months in the case of the previous migration

Reply:
the number of inferences is the same because the pruning window is the same, but the number of calculated worker scores may be less